cmd/kubeadm/app/cmd/alpha/certs.go

```diff
@@ -68,7 +68,7 @@ var (
 	This command will print out a secure randomly-generated certificate key that can be used with
 	the "init" command.
 
-	You can also use "kubeadm init --experimental-upload-certs" without specifying a certificate key and it will
+	You can also use "kubeadm init --upload-certs" without specifying a certificate key and it will
```

Comment:
Wait... This is not the only place for changes.

Since 1.15 is out, `ExperimentalUploadCerts` can be safely removed in next release and using `UploadCerts` instead.

Reply:
Thanks @dixudx  
I'll clean it up later.

Reply:
this line should have been fixed for 1.15, but we shouldn't backport non critical fixes.
for 1.16 (master) this is OK.